### PR TITLE
Make user details a page

### PR DESCRIPTION
**Pull request by dvaldivia**
This converts the user details from a modal to a page

It's part of https://github.com/miniohq/engineering/issues/117

Screen Shot 2021-05-07 at 2 20 52 PM
Screen Shot 2021-05-07 at 2 20 49 PM
Screen Shot 2021-05-07 at 2 20 45 PM
Screen Shot 2021-05-07 at 2 20 41 PM
Screen Shot 2021-05-07 at 2 20 37 PM

Signed-off-by: Daniel Valdivia <hola@danielvaldivia.com>

### CODE REVIEW

**Review by Alevsk — DISMISSED** [review]
Tested, works fine

**Review by Alevsk — DISMISSED** [review]

File: portal-ui/src/screens/shared/tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      style={{ marginTop: "5px" }}
+      {...other}
+    >
+      {value === index && <Fragment>{children}</Fragment>}
+    </div>
+  );
+};
```
Comment (bexsoft) [inline portal-ui/src/screens/shared/tabs.tsx:40]: please add export default

File: portal-ui/src/screens/Console/Users/ViewUser.tsx [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+      <Grid container>
+        <Grid item xs={12} className={classes.container}>
+          <Grid item xs={12}>
+            <Grid container spacing={2}>
+              <Grid item>
+                <Paper className={classes.paperContainer}>
+                  <div className={classes.gridContainer}>
+                    <div>Enabled:</div>
+                    <div className={classes.capitalizeFirst}>
+                      <FormSwitchWrapper
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/ViewUser.tsx:234]: Please disable this button when user is the same as the logged one

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/shared/tabs.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/shared/tabs.tsx:40]: these are shared componentes, no need for an export default

File: portal-ui/src/screens/Console/Users/ViewUser.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Users/ViewUser.tsx:234]: good pointo

Comment (dvaldivia) [reply portal-ui/src/screens/Console/Users/ViewUser.tsx:236]: done

**Review by Alevsk — APPROVED** [review]

**Pull request merged**